charts/tekton-apps/templates/wordpress/roles.yaml

```diff
@@ -16,7 +16,7 @@ kind: Role
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
  name: {{ include "tekton-apps.resourceName" (set $data "suffix" "build-pipeline-role") }}
- namespace: {{ $project.argocd.namespace }}
+ namespace: {{ include "tekton-apps.set-namespace-from-component-or-project" (dict "component" $component "project" $project "component_name" "component") }}
```

Comment:
Shouldn't we have the same changes for roles for general components? (currently we added there possibility to define own namespace, but it won't work without the stuff like in this file as I understand)

Reply:
We don't need to use the tekton-apps.set-namespace-from-component-or-project snippet in general roles.yaml  because these objects are not used in other namespaces